app/controllers/pages_controller.rb

```diff
@@ -16,4 +17,9 @@ def deploy_json
 
     render json: deploy_json
   end
+
+  # For CORS preflight requests
+  def options
+    render nothing: true
```

Comment:
codeclimate complaining this is untested. since we've now gone beyond simple config, care to make a test?